samples/Kconfig.mcumgr_bt_ota_dfu

```diff
@@ -0,0 +1,91 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+config NCS_SAMPLE_MCUMGR_BT_OTA_DFU
+	bool "MCUmgr and OTA DFU over Bluetooth"
+	select MCUMGR
+	select MCUMGR_SMP_BT
+	select MCUMGR_SMP_BT_CONN_PARAM_CONTROL
+	select MCUMGR_CMD_IMG_MGMT
+	select MCUMGR_CMD_OS_MGMT
+	select MCUMGR_GRP_ZEPHYR_BASIC
+	select MCUMGR_GRP_BASIC_CMD_STORAGE_ERASE
+	select MCUMGR_SMP_REASSEMBLY_BT
+	select NCS_SAMPLE_MCUMGR_BT_OTA_DFU_SPEEDUP
+	select OS_MGMT_MCUMGR_PARAMS
+	imply NORDIC_QSPI_NOR
+	depends on BT_PERIPHERAL
+	depends on SOC_SERIES_NRF53X
+	depends on BOOTLOADER_MCUBOOT
+	help
+	  Enable this option to include MCUmgr in the build which will enable
+	  the Bluetooth transport and image management group, which allows
+	  for firmware over the air update support of the device.
+
+if NCS_SAMPLE_MCUMGR_BT_OTA_DFU
+
+config BT_PERIPHERAL_PREF_MIN_INT
+	default 6
+
+config BT_PERIPHERAL_PREF_MAX_INT
+	default 6
+
+config BT_PERIPHERAL_PREF_LATENCY
+	default 99
+
+config BT_PERIPHERAL_PREF_TIMEOUT
+	default 400
```

Comment:
Shouldn't you actually set the parameters used during SMP data exchange? (e.g. `CONFIG_MCUMGR_SMP_BT_CONN_PARAM_CONTROL_MAX_INT`?

Reply:
@kapi-no ?

Reply:
The ```CONFIG_MCUMGR_SMP_BT_CONN_PARAM_``` values are already set to optimize for throughput. I can delete the ```BT_PERIPHERAL_PREF_```  defaults if you think they are unnecessary. I added them to maintain the previous configuration of the project that used it (for safety). We may want to remove these defaults in the post-release cleanup

Reply:
I think we could delete them if they are no longer necessary (you could double check to ensure that OTA DFU is not affected by removing them - I think it should work just fine since we enable MCUMGR_SMP_BT_CONN_PARAM_CONTROL)